buildroot-external/kernel/v6.6.y/device-support-pci.config

```diff
@@ -39,3 +39,10 @@ CONFIG_SND_HDA_CODEC_HDMI=m
 # NVMe SSD support
 CONFIG_NVME_CORE=y
 CONFIG_BLK_DEV_NVME=y
+
+# Mellanox ConnectX-3 NIC support
```

Comment:
Just a nit, but I'd move it a bit upper in the file, above `CONFIG_MEDIA_PCI_SUPPORT`, so it'd be next to other NIC configs.